frontend/views/reports.js

```diff
@@ -85,6 +133,7 @@ module.exports = Backbone.View.extend({
         this.$el.html(template(context));
         this.resultsView = new ReportTableView({
```

Comment:
Good question. I tried moving this call to the `initialize` block, but then neither the chart or table render, and I haven't yet been able to figure out why. Still trying to wrap my head around Backbone.